mitmproxy/flow_export.py

```diff
@@ -53,8 +56,16 @@ def python_code(flow):
 
     data = ""
     if flow.request.body:
-        data = "\ndata = '''%s'''\n" % flow.request.body
-        args += "\n    data=data,"
+        json_obj = is_json(flow.request.headers, flow.request.body)
+        if json_obj:
+            # Without the separators field json.dumps() produces
+            # trailing white spaces: https://bugs.python.org/issue16333
```

Comment:
The 'fix' for that issue was to update the documentation that suggests using `separators=(',', ': '))` - https://hg.python.org/cpython/rev/78bad589f205